.github/workflows/lint.yml

```diff
@@ -23,6 +23,10 @@ jobs:
         with:
           node-version: "14"
 
+      - name: Install JS dependencies
+        run: npm i
+        working-directory: ./js
+
       - name: Check JS formatting
-        run: npx prettier --check src
+        run: ./node_modules/.bin/prettier --check src
```

Comment:
This should use the package.json command, `npm run format-check`

Reply:
Ahh, I forget we have that. Updated